corehq/apps/reports/formdetails/case_metadata.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import absolute_import, unicode_literals
+
+from corehq.apps.app_manager.app_schemas.case_properties import (
```

Comment:
I'm surprised this new module (`case_metadata`) lives in `corehq.apps.reports`. Reporting modules are often leaf nodes in the dependency tree, but a case metadata builder seems like a core piece of business logic that many parts of HQ (especially things in `app_manager`) may eventually depend on. If that happens, then there will be a circular dependency between `corehq.apps.reports` and `corehq.apps.app_manager`.

I think a better place to put this new module would be in the `corehq.apps.app_manager.app_schemas`.

Reply:
I agree that this is a better place. I've moved everything into there now.